src/renderer/Core/Settings/Pages/SearchEngine/SearchEngine.tsx

```diff
@@ -28,6 +29,7 @@ export const SearchEngine = () => {
                 <RescanInterval automaticRescanEnabled={automaticRescanEnabled} />
                 <Fuzziness />
                 <MaxResultLength />
+                <MaxResultLengthEmptySearchTerm />
```

Comment:
Added to Search Engine settings, just below the existing max search results setting.